instrumentation/wicket-8.0/javaagent/build.gradle.kts

```diff
@@ -23,4 +23,7 @@ dependencies {
 
   testInstrumentation(project(":instrumentation:servlet:servlet-3.0:javaagent"))
   testInstrumentation(project(":instrumentation:servlet:servlet-javax-common:javaagent"))
+
+  // Wicket 9 requires Java 11
+  latestDepTestLibrary("org.apache.wicket:wicket:8.+")
```

Comment:
It is better to change the minimum required java version for latest dep tests instead of limiting the tested version range
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/7400025c1037e472124fc9f57d4ad64105e347c3/instrumentation/spring/spring-rabbit-1.0/javaagent/build.gradle.kts#L32-L39
Removing the `latestDepTestLibrary` line would also be fine, we don't run latest dep tests with jdk8 so this won't fail in the CI

Reply:
[testLatestDeps3](https://github.com/open-telemetry/opentelemetry-java-instrumentation/actions/runs/6867558306/job/18681444855#step:9:1649) was failing.
Anyway, I updated as suggested  to test with minimum required java version.